File: 06-JRQRCode/06-JRQRCode/ViewController.m
//
//  ViewController.m
//  06-JRQRCode
//
//  Created by jackfrow on 2019/1/11.
//  Copyright © 2019 jackfrow. All rights reserved.
//

#import "ViewController.h"
#import "WCQRCodeVC.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
   
}

- (IBAction)ShowQrcode:(id)sender {
    
    WCQRCodeVC* vc = [[WCQRCodeVC alloc] init];
    [self presentViewController:vc animated:YES completion:nil];
    
}

@end

File: 14.PhotoKitDemo/14.PhotoKitDemo/ViewController.swift
//
//  ViewController.swift
//  14.PhotoKitDemo
//
//  Created by yy on 2019/8/12.
//  Copyright © 2019 xufanghao. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
     
        
     
    }
    

    @IBAction func ShowLibrary(_ sender: Any) {
        
        JRPerMission.registerPhotoPermisson {
            
          let vc = JRPhotoViewController()
          let nav = UINavigationController(rootViewController: vc)
           self.present(nav, animated: true)
            
        }
        
    }
    
}

